faq/sql-faq.md

```diff
@@ -7,6 +7,28 @@ summary: Learn about the FAQs related to TiDB SQL.
 
 This document summarizes the FAQs related to SQL operations in TiDB.
 
+## Does TiDB support secondary key?
+
+Yes. If a column in TiDB meets all of the following conditions, it can work as a secondary key:
```

Comment:
Better not to answer this question in this way. Secondary key is not a formal terminology, but an ability to access specific row quickly by a column other than primary key.  In TiDB, we can achieve it by secondary index with not null constraint.

Reply:
addressed